airflow/www/static/js/dag/details/graph/DagNode.tsx

```diff
@@ -126,10 +125,6 @@ const DagNode = ({
           label={taskName}
           isOpen={isOpen}
           isGroup={!!childCount}
-          onClick={(e) => {
-            e.stopPropagation();
```

Comment:
We need this for opening and closing task groups. But we can wrap it in an if statement `if (!!onToggleCollapse) {...` and have both function inside. Therefore, the onclick will propagate to selecting the task if it is not a task group